src/main/java/com/github/bartimaeusnek/bartworks/system/material/GT_Enhancement/PlatinumSludgeOverHaul.java

```diff
@@ -940,6 +940,12 @@ private static boolean isInBlackList(ItemStack stack) {
             } catch (ClassNotFoundException e) {
                 e.printStackTrace();
             }
+            if (stack.getItem() == GT_ModHandler.getModItem(GTPlusPlus.ID, "itemDustHeLiCoPtEr>", 1L).getItem()) {
+                return true;
+            }
+            if (stack.getItem() == GT_ModHandler.getModItem(GTPlusPlus.ID, "itemDustWhiteMetal>", 1L).getItem()) {
```

Comment:
`>`?

Reply:
was from copy pasting mt hand. but yea I already fixed that myself :P